cpp/include/Ice/InputStream.h

```diff
@@ -177,6 +176,7 @@ namespace Ice
             b.resize(sz);
             i = b.end();
         }
+        // \endcond
```

Comment:
This needs the same number of slashes to match the `\cond` that it's closing.
```suggestion
        /// \endcond
```
Otherwise it doesn't close and the whole file is marked as `INTERNAL`.